### PR TITLE
pngext_dev

### PR DESCRIPTION
**Pull request by Pavan-Bellam**
supporting the extraction of private tags

### CODE REVIEW

**Comment by pradeeban** [comment]
Thanks.

But before I review this, can you please make sure to send your pull requests against the dev branch?

https://github.com/Emory-HITI/Niffler/blob/master/CONTRIBUTING.md

File: modules/png-extraction/ImageExtractor.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 import os
-import glob 
+import glob
```
Comment (pradeeban) [inline modules/png-extraction/ImageExtractor.py:4]: Also, each edit should have a meaniing. Please make sure these changes are not in your new pull request.

Your pull request should have the minimal changes needed for the feature, and avoid changing line breaks.

File: modules/png-extraction/ImageExtractor.py [hunk 2]
```diff
@@ -170,101 +191,80 @@ def extract_images(filedata, i, png_destination, flattened_to_level, failed, is1
...
-            try:
-                ID2=filedata.iloc[i].loc['StudyInstanceUID']  # Unique identifier for the Study.
-                ID3=filedata.iloc[i].loc['SeriesInstanceUID']  # Unique identifier of the Series.
-            except:
-                ID2='ALL-STUDIES'
-                ID3='ALL-SERIES'
-            folderName = hashlib.sha224(ID1.encode('utf-8')).hexdigest() + "/" + \
-                         hashlib.sha224(ID2.encode('utf-8')).hexdigest() + "/" + \
-                         hashlib.sha224(ID3.encode('utf-8')).hexdigest()
+    im = ds.pixel_array # pull image from read dicom
```
Comment (pradeeban) [inline modules/png-extraction/ImageExtractor.py:194]: Some of these lines do not have the actual change. Make sure only your changes are in the pull request.

This is a best practice for any open-source development and not just Emory BMI. (It helps with tracking who did the changes and also makes it easy to review).

**Comment by pradeeban** [comment]
Also, please make edits to the README too, to appropriately reflect the changes you made, to explain how to use the change.

**Comment by Pavan-Bellam** [comment]
Okay i will make those changes and make a pull request again

**Pull request closed without merging**